v3.1-RC2/gbfs.json

```
@@ -0,0 +1,70 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://github.com/MobilityData/gbfs-json-schema/blob/master/v3.1-RC2/gbfsjson",
```

@Alessandro100 Why not put a period (`.`) between the file name and json? If we put the period, the link redirects to the GitHub file, which is pretty handy, no?

```suggestion
  "$id": "https://github.com/MobilityData/gbfs-json-schema/blob/master/v3.1-RC2/gbfs.json",
```